compiler/resolution/resolveFunction.cpp

```diff
@@ -693,13 +718,9 @@ static void resolveAlsoConversions(FnSymbol* fn, CallExpr* forCall) {
     checkInitEq = false;
     checkCast = false;
 
-    // However, don't allow '=' on these types to be defined
-    // outside of the standard/internal modules
-    if (have.assign != NULL &&
-        have.assign->defPoint->getModule()->modTag == MOD_USER) {
-      USR_FATAL_CONT(have.assign->defPoint,
-                     "Can't overload assignments for class types");
-    }
+    // Checked for '=' on classes in user modules in
+    // checkForUnsupportedOverload called from resolveSignature
```

Comment:
I find this a bit hard to parse properly, I think maybe it just needs a comma after the function name?